### ISSUE TITLE
Output index with date suffix

### ISSUE THREAD

**Issue by daradermody** [regel/loudml#66]
Can Loud ML be configured to use date suffixes in the output index? The use case is where I have many models created pushing data very frequently, I would like to avoid having a very large index.

**Comment by regel** [comment]
Yes, output indexes (named `datasinks` in 1.4.3) will have this option. You only have to add * in your config.yml, eg:
```
 - name: elastic1
   type: elasticsearch
   addr: localhost:9200
   doc_type: doc
   index: bar-*
```

Writing predictions to this `datasink` will insert the date index in the output index, so eg: bar-2018.10.31.

The same rule applies to `datasources` declarations. If you include a *, then queries will fetch data from this index pattern.

Does it work for you?

**Comment by regel** [comment]
Rollover indexes, with an alias, can be an option too, if you need to limit the size / item count in the indexes: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-rollover-index.html

**Comment by daradermody** [comment]
That's perfect. Thanks, @regel.

**Comment by regel** [comment]
Hi @daradermody, do you confirm we can close this one?

**Comment by daradermody** [comment]
Yes, you can close. 1.4.3 providing this, thanks.
